packages/fxa-payments-server/src/components/PaymentConfirmation/index.scss

```diff
@@ -83,7 +83,7 @@
     background: url('./images/mastercard.svg') no-repeat;
   }
 
-  .amex::before {
+  .american.express::before {
```

Comment:
Does this need to be a chained selector? Can we get away with a single classname?

Reply:
Could just be `.american`, I suppose